tsconfig.json

```diff
@@ -1,8 +1,19 @@
 {
   "compilerOptions": {
     "target": "es2017",
-    "module": "es2015",
+    "module": "es2020",
     "moduleResolution": "node",
+    "paths": {
+      "@cloudscape-design/code-view/highlight/yaml": [
+        "./node_modules/@cloudscape-design/code-view/code-view/highlight/yaml"
+      ],
+      "@cloudscape-design/code-view/highlight/xml": [
+        "./node_modules/@cloudscape-design/code-view/code-view/highlight/xml"
+      ],
+      "@cloudscape-design/code-view/highlight/json": [
+        "./node_modules/@cloudscape-design/code-view/code-view/highlight/json"
+      ]
+    },
```

Comment:
Edited manually. The converter package doesn't take care of these imports. Will follow this up to see why.

Reply:
What is the expected behaviour from the converter package?

Reply:
Ah nevermind, you mean that we shouldn't need this at all which I agree.